src/battle_script_commands.c

```diff
@@ -8121,18 +8121,13 @@ static bool32 HasAttackerFaintedTarget(void)
 
 bool32 CanPoisonType(u8 battlerAttacker, u8 battlerTarget)
 {
-    if (GetBattlerAbility(battlerAttacker) == ABILITY_CORROSION)
-    {
-        if ((IS_BATTLER_OF_TYPE(battlerTarget, TYPE_STEEL) && (gBattleMoves[gCurrentMove].split == SPLIT_STATUS))
-        || IS_BATTLER_OF_TYPE(battlerTarget, TYPE_POISON))
-            return TRUE;
-    }
+    if (GetBattlerAbility(battlerAttacker) == ABILITY_CORROSION
+    && (IS_BATTLER_OF_TYPE(battlerTarget, TYPE_STEEL) || IS_BATTLER_OF_TYPE(battlerTarget, TYPE_POISON)))
```

Comment:
```suggestion
    if (GetBattlerAbility(battlerAttacker) == ABILITY_CORROSION
        && (IS_BATTLER_OF_TYPE(battlerTarget, TYPE_STEEL) || IS_BATTLER_OF_TYPE(battlerTarget, TYPE_POISON)))
```

*I prefer to stack the conditions personally, but most of the contributors tend to drop 4 full spaces and there's no real consistency in this regard (at least not yet), so.*

Reply:
I agree with stacking and I don't see a reason in this case to drop 4 spaces. I don't mind non stacking conditions if brackets are used but without it looks weird.

Reply:
Since only a return is used now I added 4 spaces.